<launch>
    <arg name="mass" default="1500.0" />
    <arg name="mu" default="0.8" />
    <arg name="ds" default="0.2" />
    <arg name="preview_distance" default="50" />
    <arg name="time_weight" default="0.0" />
    <arg name="smooth_weight" default="15.0" />
    <arg name="velocity_weight" default="15.0" />
    <arg name="longitudinal_slack_weight" default="1.0" />
    <arg name="lateral_slack_weight" default="10.0" />
    <arg name="curvature_weight" default="10" />
    <arg name="decay_factor" default="0.8" />
    <arg name="lateral_g" default="0.4" />
    <arg name="skip_size" default="10" />
    <arg name="smooth_size" default="50" />
    <node pkg="speed_planner" type="speed_planner" name="speed_planner" output="screen">
        <param name="mass" value="$(arg mass)" />
        <param name="mu" value="$(arg mu)" />
        <param name="ds" value="$(arg ds)" />
        <param name="preview_distance" value="$(arg preview_distance)" />
        <param name="time_weight" value="$(arg time_weight)" />
        <param name="smooth_weight" value="$(arg smooth_weight)" />
        <param name="velocity_weight" value="$(arg velocity_weight)" />
        <param name="longitudinal_slack_weight" value="$(arg longitudinal_slack_weight)" />
        <param name="lateral_slack_weight" value="$(arg lateral_slack_weight)" />
        <param name="curvature_weight" value="$(arg curvature_weight)" />
        <param name="decay_factor" value="$(arg decay_factor)" />
        <param name="lateral_g" value="$(arg lateral_g)" />
        <param name="skip_size" value="$(arg skip_size)" />
        <param name="smooth_size" value="$(arg smooth_size)" />
    </node>
</launch>
